job-server/test/spark.jobserver/WebApiMainRoutesSpec.scala

```diff
@@ -352,6 +352,9 @@ class WebApiMainRoutesSpec extends WebApiSpec {
       Post("/contexts/meme?num-cpu-cores=3&coarse-mesos-mode=true") ~> sealRoute(routes) ~> check {
         status should be (OK)
       }
+      Post("/contexts/meme?num-cpu-cores=3", "spark.context-settings.test=1") ~> sealRoute(routes) ~> check {
```

Comment:
Merged, thanks again!

> On Jul 1, 2016, at 8:16 AM, Casey Green <notifications@github.com> wrote:
> 
> In job-server/test/spark.jobserver/WebApiMainRoutesSpec.scala <https://github.com/spark-jobserver/spark-jobserver/pull/531#discussion_r69314731>:
> 
> > @@ -352,6 +352,9 @@ class WebApiMainRoutesSpec extends WebApiSpec {
> >        Post("/contexts/meme?num-cpu-cores=3&coarse-mesos-mode=true") ~> sealRoute(routes) ~> check {
> >          status should be (OK)
> >        }
> > +      Post("/contexts/meme?num-cpu-cores=3", "spark.context-settings.test=1") ~> sealRoute(routes) ~> check {
> After understanding the flow a bit more, this is another approach to testing this: https://github.com/casey-green/spark-jobserver/compare/casey/context-spark-config-in-body...casey-green:casey/context-spark-config-in-body-alt-test-approach?expand=1 <https://github.com/casey-green/spark-jobserver/compare/casey/context-spark-config-in-body...casey-green:casey/context-spark-config-in-body-alt-test-approach?expand=1>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/spark-jobserver/spark-jobserver/pull/531/files/b3273e2ca7426558793121db90fa4255b686ee97#r69314731>, or mute the thread <https://github.com/notifications/unsubscribe/ABA3226cAHjiRBy63i2h_zlWt5VdvWQmks5qRS9IgaJpZM4JAo6W>.
>